azext_hanaonazure/modules_sdk/models/_models.py

```diff
@@ -506,9 +506,18 @@ class SapMonitor(Resource):
     :ivar managed_resource_group_name: The name of the resource group the SAP
      Monitor resources get deployed into.
     :vartype managed_resource_group_name: str
-    :ivar log_analytics_workspace_arm_id: The ARM ID of the Log Analytics
+    :param log_analytics_workspace_arm_id: The ARM ID of the Log Analytics
      Workspace that is used for monitoring
-    :vartype log_analytics_workspace_arm_id: str
+    :type log_analytics_workspace_arm_id: str
```

Comment:
Have all these been auto generated because of the new log analytics params?

Reply:
yup, everything in that directory is generated